src/components/pages/Give/GiveForm.tsx

```diff
@@ -0,0 +1,328 @@
+import { Big } from "big.js";
+import * as React from "react";
+import { Button, View, Text, TextInput, StyleSheet } from "react-native";
+import { connect, MapStateToProps, MergeProps } from "react-redux";
+
+import { ApiEndpoint } from "../../../api";
+import { MemberId } from "../../../identifiers";
+import { RahaState } from "../../../store";
+import { give } from "../../../store/actions/wallet";
+import {
+  ApiCallStatus,
+  ApiCallStatusType
+} from "../../../store/reducers/apiCalls";
+import { Member } from "../../../store/reducers/members";
+import { getStatusOfApiCall } from "../../../store/selectors/apiCalls";
+import { getLoggedInMember } from "../../../store/selectors/authentication";
+import { getMembersByIds } from "../../../store/selectors/members";
+import { MemberSearchBar } from "../../shared/MemberSearchBar";
+
+const MESSAGE_PLACEHOLDER_TEXT = "";
+const MAX_MEMO_LENGTH = 140;
+
+type OwnProps = {
+  toMemberId?: MemberId;
+  identifier: string;
+  onSuccessCallback: (toMember: Member, amount: string, memo: string) => any;
```

Comment:
Please leave a doc comment explaining what `identifier` and `onSuccessCallback` are, as it's not immediately apparent